src/actions/traderAction.js

```diff
@@ -1,4 +1,12 @@
 import { FETCH_TRADERS_BY_TOKEN, FETCH_TRADERS_START, FETCH_TRADERS_ERROR } from './actionTypes';
+import Web3 from 'web3';
+import abi from '../constants/abi';
+
+const INFURA_ENDPOINT = 'https://mainnet.infura.io/v3/a809da2620ae4733aa86edde64cc86ad';
+const NECTAR_CONTRACT_ADDRESS = '0xcc80c051057b774cd75067dc48f8987c4eb97a5e';
```

Comment:
Please replace both of these endpoints by reading from https://github.com/ethfinex/community-gateway/blob/master/src/constants/config.dist.json

During the deployment flow on production, `config.dist.json` is copied to `config.json` the config variables are read from `config.json`.

This will also ensure it works correctly on kovan (where we use config.dev.json).

Both infura url and nectar address are in there.